experimental/web_dashboard/test/mock_service_test.dart

```diff
@@ -0,0 +1,93 @@
+import 'package:test/test.dart';
```

Comment:
nit: Copyright header.

Side note: you have no idea how happy it makes me to see tests going into one of our samples while it's being coded. :)